datadog_lambda/wrapper.py

```diff
@@ -190,6 +194,18 @@ def _after(self, event, context):
             status_code = extract_http_status_code_tag(self.trigger_tags, self.response)
             if status_code:
                 self.trigger_tags["http.status_code"] = status_code
+                if len(status_code) == 3 and status_code.startswith("5"):
+                    submit_errors_metric(context)
+                    if self.span:
+                        self.span.set_traceback()
+                        self.span.error = 1
+                        self.span.set_tags(
+                            {
+                                ERROR_TYPE: "5xx Server Errors",
+                                ERROR_MSG: SERVER_ERRORS_STATUS_CODES.get(
+                                    status_code, "5xx Server Errors"),
```

Comment:
I wonder if we should set `error.trace` here as some hard-coded string since the stack trace of `span.set_traceback()` is not helpful and may be confusing.

<img width="1331" alt="image" src="https://user-images.githubusercontent.com/47579703/175381062-5b54ab64-4c21-44d3-b693-0113037be286.png">